postfix/values.yaml

```diff
@@ -806,6 +806,20 @@ postfix:
     # sasl_passwd: |-
     #   [smtp.sendgrid.net]:587 yourSendGridUsername:yourSendGridPassword
 
+metrics:
+  enabled: false
+  image:
+    repository: chatwork/kumina-postfix-exporter
+    tag: k-kinzal-fixes-panic-logfilepath-empty
+    pullPolicy: IfNotPresent
+  command:
+    - /bin/postfix_exporter
+    - --postfix.logfile_path=""
```

Comment:
The default postfix image for this chart does not have log rotates.
For this reason, exporter disables the logging feature.